docker/README.md

```diff
@@ -17,19 +17,37 @@ The `fiftyone-teams-app`, `fiftyone-teams-api`, and `fiftyone-app` images are av
 
 ## Initial Installation vs. Upgrades
 
-`FIFTYONE_DATABASE_ADMIN` is set to `false` by default for FiftyOne Teams version 1.2.1 installations and upgrades. This is because FiftyOne Teams version 1.2.1 is backwards compatible with FiftyOne Teams database schema version 0.19 (Teams Version 1.1).
+`FIFTYONE_DATABASE_ADMIN` is set to `false` by default for FiftyOne Teams version 1.3.0 installations and upgrades. This is because FiftyOne Teams version 1.3.0 is backwards compatible with FiftyOne Teams database schema version 0.19 (Teams Version 1.1) and newer.
 
-- If you are performing an initial install, you will either want to connect to your MongoDB database with the 0.12.1 SDK before performing the FiftyOne Teams installation, or you will want to set `FIFTYONE_DATABASE_ADMIN: true` in the `environment` section of the `fiftyone-app` service definition.
+- If you are performing an initial install, you will either want to connect to your MongoDB database with the 0.13.0 SDK before performing the FiftyOne Teams installation, or you will want to set `FIFTYONE_DATABASE_ADMIN: true` in the `environment` section of the `fiftyone-app` service definition.
 
 - If you are performing an upgrade, please review our [Upgrade Process Recommendations](#upgrade-process-recommendations)
 
 ---
 
 ### FiftyOne Teams Upgrade Notes
 
+#### Enabling FiftyOne Teams Plugins
+
+FiftyOne Teams v1.3.0 officially introduces [Plugins](https://docs.voxel51.com/plugins/index.html) to customize and extend the functionality of FiftyOne Teams in your environment.  There are three modes for plugins:
+
+- Builtin Plugins Only - no changes are required for this mode.
+- Plugins run in the `fiftyone-app` deployment - to enabled this mode:
+    - you should use the `compose.plugins.yaml` instead of the `compose.yaml` in this repository.  Your `docker compose` command might look like:<br>
+        `docker compose -f compose.plugins.yaml -f compose.override.yaml up -d`
+    - expose the `teams-api` for SDK access
```

Comment:
maybe add a link the instructions [docker/expose-teams-api.md](https://github.com/voxel51/fiftyone-teams-app-deploy/pull/26/files#diff-1c5554712a1ee001cb74a2d260b7086731bd98fdefe310a75e022d7189fba35e)